spec/spec_helper.rb

```diff
@@ -37,8 +37,8 @@ def create_database
   ActiveRecord::Schema.define(:version => 1) do
     create_table :users do |t|
       t.string :name
-      t.string :colors, array: true, defualt: '{}'
-      t.timestamps
+      t.string :colors, array: true, default: []
```

Comment:
Should you update the Readme too?   https://github.com/psylone/acts-as-taggable-array-on/blob/1a7a9db771f44e0b62701152354035bf62964bb9/README.md#setup